company/index.html

```diff
@@ -54,11 +54,13 @@
                   </p>
                   {{member.fullDescription}}
                   <ul class="list-unstyled list-inline team-socail">
+                    {% if member.twitter != null %}
```

Comment:
May be better to have these in a separate PR, what do you think ? To not mix content changes with the behaviour changes

Reply:
Okay, I'll put this change in a separate PR and we'll need to merge that one first.

Reply:
cool, you can say so in the description and reference them both using #763 notation